src/main.js

```diff
@@ -116,14 +118,36 @@ var currentPoster;
 
 // event listeners go here 👇
 
+showMyPosterButton.addEventListener('click', function(e) {
```

Comment:
As a refactor I am wondering if we want to pass the event object in the designPoster function instead of on this event listener. I'm wondering what best practice might be.